templates/user/auth/twofa_scratch.tmpl

```diff
@@ -17,6 +17,7 @@
 					<div class="inline field">
 						<label></label>
 						<button class="ui green button">{{.i18n.Tr "auth.verify"}}</button>
+						 <a href="{{AppSubUrl}}/user/two_factor">{{.i18n.Tr "use_passcode" | Str2html}}</a>
```

Comment:
Is `Str2html` needed here? As this string is just a string I don't think escaping needs to be used.

Reply:
i see the twofa.tmpl page's button has Str2html.
i think the functions of the two buttons are the same.
so i added it. i'm not sure,is it necessary

Reply:
I think it is ok not to have it because there is no HTML in the translation string.